### ISSUE TITLE
PDFs made with MS Edge don't parse

### ISSUE THREAD

**Issue by nadkab11** [smalot/pdfparser#726]
Steps to reproduce:
-open both Chrome and Edge browsers
-go to example.com in each one
-convert that page to PDF in each browser

compare the two pdf files. the one from MS Edge won't parse.

**Comment by k00ni** [comment]
Just to be clear, "won't parse" means no result? Or an error?

**Comment by nadkab11** [comment]
Yes, its an empty result for this line: $text = $pdf->getText();

Is there a debug mode or way to see errors?
